decorator.py

```diff
@@ -0,0 +1,36 @@
+"""
+This task contains a few parts:
+Write a custom function which returns your name or username.
+Write a decorator which allows access for your user and prints 
+�Permission denied� for other users.
+
+Tips:
+Create get_user_info function which returns your name or username.
+Create decorator which raises Exception if get_user_info returns not 
+your name or username
+Create check_perms function decorated by decorator. This function
+returns Allowed access if user validation is successful.
+"""
+
+
+def decorator(f):
+    def wrapper(my_name):
+        name_list = ["Sergiy Budz", "Pavlo"]
```

Comment:
We don't need any hardcoded values in our decorator.

Reply:
@Sergiy-1998  ?

Reply:
done! look at the last commit

Reply:
I don't need to look at any commit.
You should respond to EVERY comment.

Reply:
all comments are closed